sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/CqlDataWriterDAO.java

```diff
@@ -265,19 +267,20 @@ public void writeRows(String placementName, Iterator<MigrationScanResult> iterat
         DeltaPlacement placement = (DeltaPlacement) _placementCache.get(placementName);
         Session session = placement.getKeyspace().getCqlSession();
         BatchStatement statement = new BatchStatement(BatchStatement.Type.LOGGED);
-        final Semaphore semaphore = new Semaphore(maxConcurrentWrites);
         AtomicReference<Throwable> error = new AtomicReference<>();
+        RateLimiter rateLimiter = RateLimiter.create(1000);
```

Comment:
Shouldn't this be `maxWritesPerSeconds` instead of 1000?

Reply:
yes. whoops